lib/bootloader_pvm.pm

```diff
@@ -67,6 +67,90 @@ sub get_into_net_boot {
+    assert_screen [qw(pvm-poweroff-successful pvm-poweroff-not-running)], 180;
+
+    # proceed with normal boot if is system already installed, use sms boot for installation
+    my $bootmode = get_var('BOOT_HDD_IMAGE') ? "norm" : "sms";
+    type_string "chsysstate -r lpar -m $hmc_machine_name -o on -b ${bootmode} --id $lpar_id && echo 'LPAR SUCCESSFULLY BOOTED'\n";
+    assert_screen "pvm-poweron-successful";
+
+    # don't wait for it, otherwise we miss the menu
+    type_string "mkvterm -m $hmc_machine_name --id $lpar_id\n";
+    get_into_net_boot;
+
+    # the grub on powerVM has a rather strange feature that it will boot
+    # into the firmware if the lpar was reconfigured in between and the
+    # first menu entry was used to enter the command line. So we need to
+    # reset the LPAR manually
+    if (match_has_tag('novalink-failed-first-boot')) {
+        type_string "set-default ibm,fw-nbr-reboots\n";
+        type_string "reset-all\n";
+        assert_screen 'pvm-firmware-prompt';
+        send_key '1';
+        get_into_net_boot;
+    }
+    # try 3 times but wait a long time in between - if we're too eager
+    # we end with ccc in the prompt
+    send_key_until_needlematch('pvm-grub-command-line', 'c', 3, 5);
+
+    # clear the prompt (and create an error) in case the above went wrong
+    type_string "\n";
+
+    my $repo     = get_required_var('REPO_0');
+    my $mirror   = get_netboot_mirror;
+    my $mntpoint = "mnt/openqa/repo/$repo/boot/ppc64le";
+    assert_screen "pvm-grub-command-line-fresh-prompt", no_wait => 1;
+    type_string_slow "linux $mntpoint/linux vga=normal install=$mirror ";
+    bootmenu_default_params;
+    bootmenu_network_source;
+    specific_bootmenu_params;
+    registration_bootloader_params(utils::VERY_SLOW_TYPING_SPEED);
+    type_string_slow remote_install_bootmenu_params;
+    type_string_slow "\n";
+
+    assert_screen "pvm-grub-command-line-fresh-prompt", 180, no_wait => 1;    # kernel is downloaded while waiting
+    type_string_slow "initrd $mntpoint/initrd\n";
+
+    assert_screen "pvm-grub-command-line-fresh-prompt", 180, no_wait => 1;    # initrd is downloaded while waiting
+    type_string "boot\n";
+    save_screenshot;
+
+    assert_screen("novalink-successful-first-boot", 120);
+    assert_screen("run-yast-ssh",                   120);
+
+    # Delete partition table before starting installation
+    select_console('install-shell');
+    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
+        script_run("wipefs -a /dev/$disk");
+        # For cryptlvm+activate existing scenario, create empty enrypted partition
+        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+    }
+    # Switch to installation console (ssh or vnc)
+    select_console('installation');
+    # We need to start installer only if it's pure ssh installation
+    type_string("yast.ssh\n") if get_var('VIDEOMODE', '') =~ /ssh-x|text/;
+    wait_still_screen;
```

Comment:
this is wasting at least 7 seconds. Do we actually need it? What are you waiting for? If you really want to have a `wait_still_screen` please consider a lower than default stilltime, e.g. `wait_still_screen(1)`

Reply:
nah… I'm not touching this. It was already there and if it is really such a huge deal feel free to open another PR afterwards. Obviously in the past it was good enough to meet our merge criteria.